A Brazilian Portuguese translation to Oreganized

Bea-CEO opened the pull request:
I made a pt_br translation for Oreganized. I hope you guys like it.

Bea-CEO committed:
Add files via upload

pt_br translation to Oreganized

PssbleTrngle commented:
thank you, you will need to move it into the same folder under `main` instead of `generated`  tho.
The files in there are automatically generated by code and it will be deleted the next time then

Bea-CEO commented:
> ok, thanks for the advice lmao, I'm gonna move it to the right folder (and also update the file because of the devices drop)

Bea-CEO committed:
Updated pt_br Translation

Remember that portuguese translation I made a while ago (which turned out to be the wrong folder)?

Well, first off: Now it's in the right folder, and second off: Now it's updated to the devices drop.

Bea-CEO committed:
Updated pt_br translation

Remember that portuguese translation I made a while ago (which turned out to be in the wrong folder)?

Well, first off: Now it's in the right folder, and second off: Now it's updated to the devices drop.

Bea-CEO commented:
> thank you, you will need to move it into the same folder under `main` instead of `generated` tho. The files in there are automatically generated by code and it will be deleted the next time then

Now It's updated and in the right folder (and I accidentally sent it twice, so just ignore the 2nd one lmao)

PssbleTrngle commented:
Thanks I can't really ignore the other file, since that's not how pull requests work, but I'll just remove it afterwards, thanks ❤️
